ERCS/erc-7647.md

```diff
@@ -0,0 +1,157 @@
+
+By combining [ERC-20](./eip-20.md) and [ERC-721](./eip-721.md) functionalities into a single token standard, developers can streamline their smart contract architecture and simplify interactions for users who hold both types of assets. This proposal aims to address this need by defining a unified standard for hybrid tokens on the Ethereum blockchain.
+
+Furthermore, there are liquidity issues associated with the auction model of the [ERC-721](./eip-721.md) standard and this proposal aims to address some of those limitations present in the NFT marketplace by allowing NFT's to be sold in whole and in pieces through a Decentralized Exchange (DEX).
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+
+The E741 token standard MUST inherit all functions and events from both [ERC-20](./eip-20.md) and [ERC-721](./eip-721.md), with additional considerations for interoperability between the two types of tokens. It is RECOMMENDED to support both types of events through an internal library, however a token MAY use another interface if it achieves the same event structure. 
+
+It is RECOMMENDED to support optional metadata features from  [ERC-20](./eip-20.md) and [ERC-721](./eip-721.md).
+
+The following rules MUST be followed to adhere to the E741 standard:
+
+- `1.0` [ERC-20](./eip-20.md) tokens represent `1` NFT.
+- The decimals of the token combined with `1.0` represent the token id.
+      -i.e. 1.12345 would represent token id 12345
+- The maximum total supply corresponds to the total number of NFTs sans decimal. It is OPTIONAL to either fix the total supply with the maximum NFT id, or to increase the total supply as new NFTs are minted.
+     - i.e. If the largest NFT id is 123, the maximum total supply is `123`.
+- The decimals of the token MUST accomodate the number of posisble NFTs.
+     - i.e. Four decimals allows for max NFT id of 9999
+- Specific ids can be transfered by using their [ERC-20](./eip-20.md) representation
+     - i.e. `transfer(to, 1.12345)` would send token id 12345
+- Sending a specifc token id will only transfer `1.0` [ERC-20](./eip-20.md) tokens along side the nft.
+     - i.e. `transfer(to, 1.12345)` would send token id 12345 + `1.0` [ERC-20](./eip-20.md) tokens
+- NFT's will be 'broken' during a transfer that forces a user's balance to decrease in such a way as to cross a `1.0` [ERC-20](./eip-20.md) threashold.
+- NFT's will be 'unbroken' during a transfer that forces a user's balance to increase in such a way as to cross a `1.0` [ERC-20](./eip-20.md) threshold.
+- NFT's that are 'broken' must be maintained in a 'broken list' temporarily.
+
+
+### Interface
+
+```solidity
```

Comment:
Thank you for the feedback.
The library is required to fully and completely adhere to both the ERC20 and ERC721 specs, so I hope this would be a case where it is allowed.

Reply:
If needed I can elaborate on why the library is technically needed.